roles/nginx/templates/wordpress.conf

```diff
@@ -1,31 +1,48 @@
+# WordPress single site rules.
+# https://www.nginx.com/resources/wiki/start/topics/recipes/wordpress/
+# Designed to be included in any server {} block.
+
+# Upstream to abstract backend connection(s) for php
+upstream php {
+        server unix:/var/run/php-fpm/wordpress.sock;
+        server 127.0.0.1:9000;
+}
+
 server {
-    listen       443 default_server;
-    server_name  fossrit.jwf.io;
-    root         /var/www/wordpress/;
+        server_name     grantgreen.justinwflory.com;
+        root            /var/www/html;
+        index           index.html index.php;
+
+        location = /favicon.ico {
+            log_not_found   off;
+            access_log      off;
+        }
 
-	client_max_body_size 64M;
```

Comment:
Are you sure your want to remove the caching? I don't care personally, but just want to make sure you didn't do this by accident since you have some more caching stuff below.

Reply:
@ct-martin Actually I'm not sure how this one snuck in there. I'll go ahead and remove it. Thanks for catching!